packages/react/.storybook/_container.scss

```diff
@@ -44,6 +44,7 @@ body {
   background-color: $ui-background;
   color: $text-01;
   height: 100%;
+  word-break: keep-all;
```

Comment:
Can we make it Korean language-only?

Reply:
When testing I believe it can't detect the language unless the specific tag is specifically set like `<p lang="ko">한</p>`. This then will not change if the a storybook user just tries typing within the knobs.

Reply:
@ariellalgilmore Would you try [`:lang(ko)`](https://developer.mozilla.org/en-US/docs/Web/CSS/:lang? )?

Reply:
Yeah I tried `  lang(ko) {
    word-break: keep-all;
  }`, but nothing changes in storybook when then typing Korean into the storybook knobs

Reply:
Sorry I think I made a typo in the issue comment. I meant one beginning with `:`. Would you try?

Reply:
I tried both options

Reply:
Would you want to try to take a deeper look at https://developer.mozilla.org/en-US/docs/Web/CSS/:lang and see how the selector supposed to work? Or would you try to investigate how to make the style change Korean-only? We need to take more time for investigation. Feel free to ping me for any questions. Thanks!